SPECS/rubygem-ffi/rubygem-ffi.spec

```diff
@@ -1,18 +1,26 @@
+# Disabled debuginfo package as the autogenerated 'debugfiles.list' file is empty.
+# In other words there were no debug symbols to package.
 %global debug_package %{nil}
 %global gem_name ffi
 Summary:        Ruby FFI
 Name:           rubygem-ffi
-Version:        1.15.5
-Release:        2%{?dist}
+Version:        1.16.3
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Languages
 URL:            https://github.com/ffi/ffi
-Source0:        https://github.com/ffi/ffi/archive/refs/tags/v%{version}.tar.gz#/%{gem_name}-%{version}.tar.gz
-Patch0:         fix-file_list.patch
+# Using the source code from the gem to include the 'libffi' submodule.
+# The upstream depends on the existence of the 'libffi' submodule, even
+# if the system's version of 'libffi' ends up being used.
+# Submodules are not included in GitHub release tarballs.
+Source0:        https://rubygems.org/gems/%{gem_name}-%{version}.gem
```

Comment:
Would check with @jslobodzian to see if we're ok using pre-built gems as sources

Reply:
We're not using any pre-built binaries from there. Gems just also contain the original sources, with which they were built.

Reply:
But thank you for pointing this out. When we were working on the `rubygem` packages I didn't know about gems containing sources as well and I think we did a bunch of unnecessary work because of that.